ui/src/shared/components/InputClickToEdit.js

```diff
@@ -88,7 +96,8 @@ const {func, bool, number, string} = PropTypes
 InputClickToEdit.propTypes = {
   wrapperClass: string.isRequired,
```

Comment:
You could pull `<div className={wrapperClass}>` outside the if statement in the render function to stay DRY. :)